address.go

```diff
@@ -83,13 +83,21 @@ func (f Format) CheckRegion(region string) bool {
 //
 // An empty postal code is considered valid.
 func (f Format) CheckPostalCode(postalCode string) bool {
-	if postalCode == "" || f.PostalCodePattern == "" {
```

Comment:
A PostalCodePattern tends to be empty only if a format does not have a postalCode field at all. In which case the current check might be enough? Or do we gain anything by compiling a "^.+$" regexp and matching the postalCode against it in that case?

Reply:
Thanks for the context! Not sure if it's good to report `true` when the format doesn't require a postal code and the input is a non empty string. Probably `^$` regex would make more sense in that case as then you can detect that there's something unexpected provided.